src/test/java/me/karakelley/http/helpers/TempFilesHelper.java

```diff
@@ -31,9 +31,9 @@ public static void withTempDirectory(Consumer<Path> f) {
     }
   }
 
-  public static Path createTempFile(Path directory, String name) {
+  public static Path createTempFile(Path directory, String name, String extension) {
```

Comment:
Any particular reason you made the extension a third argument instead of just appending it to the `name` argument (i.e. `TempFilesHelper.createTempFile(directory, "/test1.txt")`)?
OR, you could overload `createTempFile` to default to `.txt` when only two args are provided, or provide a third extension argument.

It just seems a little strange to have to specify `".txt"` in all these tests that don't seem to actually use/care about it.